lib/v2/lisa/radar.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    'sonymusic.co.jp': {
+        _name: 'Sony Music',
+        www: [
+            {
+                title: 'LiSA News',
+                docs: 'https://docs.rsshub.app/other.html#lisa',
+                source: ['/artist/lisa/'],
+                target: '/lisa/info',
+            },
+            {
+                title: 'LiSA Albums',
+                docs: 'https://docs.rsshub.app/other.html#lisa',
+                source: ['/artist/lisa/'],
+                target: '/lisa/disco',
+            },
+        ],
+    },
+    'lxixsxa.com': {
+        _name: 'LiSA Official',
+        www: [
+            {
+                title: 'LiSA News',
+                docs: 'https://docs.rsshub.app/other.html#lisa',
```

Comment:
It should point to `/other.html`

Reply:
Thank you for your review.
I have mistaken all the docs link, I will change them all to `https://docs.rsshub.app/en/live.html#lisa`, which is the doc page url of lisa router when being merged. Do I fix them correctly?

Reply:
Yes